interfaces/Glitter/templates/static/javascripts/glitter.main.js

```diff
@@ -134,7 +134,7 @@ function ViewModel() {
     })
 
     self.updateCheckAllButtonState = function(section) {
-        setCheckAllState('.multioperations-selector #multiedit-checkall', `.${section}-table input[name="multiedit"]`)
+        setCheckAllState(`[data-section="${section}"] #multiedit-checkall`, `.${section}-table input[name="multiedit"]`)
```

Comment:
```suggestion
        setCheckAllState('[data-section="${section}"] #multiedit-checkall', '.${section}-table input[name="multiedit"]')
```
I think you are on a mac and the quotes got converted!

But maybe there's a better way? We now have duplicate ID's: `multiedit-checkall`. Why not make that `multiedit-checkall-queue` and `multiedit-checkall-history` and use that to select on instead of adding `data-section`?

Reply:
You're right, for some reason I treated `multiedit-checkall` like a class instead of id, my bad.
The quotes thing is necessary because of string interpolation.